build/appsecrets.sh

```diff
@@ -0,0 +1,20 @@
+#REMEMBER TO SETUP PIPELINE VARIABLE
+# TwitchAPIKey
+
+if [ ! -n $(TwitchAPIKey) ]
+then
+    echo "You need define the CustomVisionProjectId variable in VSTS"
```

Comment:
The variable name in echo should match TwitchAPIKey

Reply:
Good catch! fixed!